### ISSUE TITLE
Pre-work Checklist: Developer: Mai Vang

### ISSUE THREAD

**Issue by vmaineng** [hackforla/website#3683]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address with a merge team member or technical lead so you can be added to the Google Drive
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [ ] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [ ] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [ ] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by ExperimentsInHonesty** [comment]
@vmaineng I see you had not self-assigned yourself to this issue yet, so I assigned you.  Please update the role label to front end, back end, or both.

**Comment by ExperimentsInHonesty** [comment]
@vmaineng Thanks for adding the labels.  I see you have not self assigned a good first issue yet.  We are formally on break until January, but you can go ahead and try to get started so that you know where you need help when the team comes back.

Here is where you can find the issues: [project board. front end, good first issue](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22good+first+issue%22#column-7198257)

**Comment by vmaineng** [comment]
Thanks for letting me know, Bonnie @ExperimentsInHonesty . I appreciate it!

**Comment by vmaineng** [comment]
Progress: "What is the current status of your project? What have you completed and what is left to do?"

I am working on [Remove unused alt-hero in project md files](https://github.com/hackforla/website/issues/2722#top)
#2722. I have reviewed and verified the only md files that has 'alt-hero' is in project-md-file--removing-unused--alt-hero--field.md. 

Blockers: "Difficulties or errors encountered."
I am going to ask the merge team if this md file should be flagged. If not, I will submit a new commit and close the issue.

Availability: "How much time will you have this week to work on this issue?"
As long as it is needed.

ETA: "When do you expect this issue to be completed?"
It depends on how fast the merge team responds back to me. 

Pictures (optional): "Add any pictures of the visual changes made to the site so far."

**Comment by vmaineng** [comment]
Progress: "What is the current status of your project? What have you completed and what is left to do?"

I am working on https://github.com/hackforla/website/issues/2722#top
https://github.com/hackforla/website/issues/2722. I have reviewed and verified the only md files that has 'alt-hero' is in project-md-file--removing-unused--alt-hero--field.md.

Blockers: "Difficulties or errors encountered."
The merge team has not responded back to me if the md file should be flagged.

Availability: "How much time will you have this week to work on this issue?"
As long as it is needed.

ETA: "When do you expect this issue to be completed?"
Still waiting for the merge team to respond back to me.

**Comment by ExperimentsInHonesty** [comment]
@vmaineng the progress update needs to go on the issue you are working on.  So if the blocker is on issue 2722, the comment should go there.

**Comment by vmaineng** [comment]
Progress: "What is the current status of your project? What have you completed and what is left to do?"
I am working on #2276.

Blockers: "Difficulties or errors encountered."
The blocker I encountered was ensuring program-area was rendering.

Availability: "How much time will you have this week to work on this issue?"
An hour

ETA: "When do you expect this issue to be completed?"
It is completed and a pull request has been submitted. 

Pictures (optional): "Add any pictures of the visual changes made to the site so far."

**Comment by vmaineng** [comment]
Progress: "What is the current status of your project? What have you completed and what is left to do?"
I am working on Redesigned homepage's CoP section img tag: Add alt property and remove the / #3792

Blockers: "Difficulties or errors encountered."
The blocker I encountered was ensuring I created the correct wiki branch

Availability: "How much time will you have this week to work on this issue?"
An hour

ETA: "When do you expect this issue to be completed?"
I plan to have it completed by tomorrow night after the developer's meeting.

**Comment by ExperimentsInHonesty** [comment]
@vmaineng I think you got the hang of doing updates, so I am going to close this issue as completed.  The next step for you is to self assign a [large front end](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+large%22#column-7198257) issue

**Comment by HackforLABot** [comment]
## Activity Log: vmaineng
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- vmaineng opened: https://github.com/hackforla/website/issues/3683 at 10/31/2022, 07:14 PM PDT
- vmaineng assigned: https://github.com/hackforla/website/issues/3683 at 11/26/2022, 11:18 PM PST
- vmaineng commented on: https://github.com/hackforla/website/issues/3683#issuecomment-1335872594 at 12/02/2022, 01:44 PM PST
- vmaineng commented on: https://github.com/hackforla/website/issues/3683#issuecomment-1356883986 at 12/18/2022, 01:53 PM PST
- vmaineng commented on: https://github.com/hackforla/website/issues/3683#issuecomment-1370252170 at 01/03/2023, 01:41 PM PST
- vmaineng assigned: https://github.com/hackforla/website/issues/2276#issuecomment-926948199 at 01/13/2023, 09:31 AM PST
- vmaineng opened: https://github.com/hackforla/website/pull/3814 at 01/13/2023, 12:14 PM PST
- vmaineng commented on: https://github.com/hackforla/website/issues/3683#issuecomment-1382344592 at 01/13/2023, 12:17 PM PST
- vmaineng closed as completed: https://github.com/hackforla/website/issues/3683#event-8224121808 at 01/13/2023, 12:17 PM PST
- vmaineng commented on: https://github.com/hackforla/website/issues/2276#issuecomment-1382345547 at 01/13/2023, 12:18 PM PST
- vmaineng reopened: https://github.com/hackforla/website/issues/3683#event-8224121808 at 01/13/2023, 12:25 PM PST
- vmaineng commented on: https://github.com/hackforla/website/pull/3814#issuecomment-1384294664 at 01/16/2023, 08:29 AM PST
- vmaineng commented on: https://github.com/hackforla/website/pull/3814#issuecomment-1384637272 at 01/16/2023, 02:41 PM PST
- vmaineng PR was merged: https://github.com/hackforla/website/pull/3814#event-8322185100 at 01/20/2023, 06:54 AM PST
- vmaineng issue closed by PR 3814: https://github.com/hackforla/website/issues/2276#event-8322185418 at 01/20/2023, 06:54 AM PST
- vmaineng assigned: https://github.com/hackforla/website/issues/3792 at 01/23/2023, 02:46 PM PST
- vmaineng commented on: https://github.com/hackforla/website/issues/3792#issuecomment-1401210621 at 01/23/2023, 04:40 PM PST
- vmaineng commented on: https://github.com/hackforla/website/issues/3683#issuecomment-1401217083 at 01/23/2023, 04:48 PM PST
- vmaineng opened: https://github.com/hackforla/website/pull/3871 at 01/24/2023, 05:20 PM PST
- vmaineng commented on: https://github.com/hackforla/website/issues/3792#issuecomment-1402917296 at 01/24/2023, 05:21 PM PST
- vmaineng PR was closed: https://github.com/hackforla/website/pull/3871#event-8369854758 at 01/26/2023, 08:00 PM PST
- vmaineng opened: https://github.com/hackforla/website/pull/3883 at 01/26/2023, 08:11 PM PST
- vmaineng commented on: https://github.com/hackforla/website/issues/3792#issuecomment-1405995215 at 01/26/2023, 08:12 PM PST
- vmaineng commented on: https://github.com/hackforla/website/pull/3883#issuecomment-1409623658 at 01/30/2023, 05:48 PM PST
- vmaineng PR was merged: https://github.com/hackforla/website/pull/3883#event-8406528988 at 01/31/2023, 09:26 PM PST
- vmaineng issue closed as completed: https://github.com/hackforla/website/issues/3792#event-8406555552 at 01/31/2023, 09:32 PM PST
- vmaineng commented on: https://github.com/hackforla/website/pull/3907#issuecomment-1412394029 at 02/01/2023, 08:58 AM PST
- vmaineng commented on: https://github.com/hackforla/website/pull/3907#issuecomment-1412685266 at 02/01/2023, 12:32 PM PST
- vmaineng submitted review: https://github.com/hackforla/website/pull/3907#pullrequestreview-1279827882 at 02/01/2023, 12:37 PM PST
- vmaineng assigned: https://github.com/hackforla/website/issues/3110#issuecomment-1414615124 at 02/12/2023, 04:50 PM PST
- vmaineng commented on: https://github.com/hackforla/website/issues/3110#issuecomment-1427186603 at 02/12/2023, 04:52 PM PST
- vmaineng commented on: https://github.com/hackforla/website/issues/3110#issuecomment-1430318210 at 02/14/2023, 12:14 PM PST
- vmaineng opened: https://github.com/hackforla/website/pull/3978 at 02/14/2023, 05:51 PM PST
- vmaineng commented on: https://github.com/hackforla/website/issues/3110#issuecomment-1430639780 at 02/14/2023, 05:53 PM PST
- vmaineng commented on: https://github.com/hackforla/website/pull/3978#issuecomment-1433354522 at 02/16/2023, 08:23 AM PST
- vmaineng PR was merged: https://github.com/hackforla/website/pull/3978#event-8542861532 at 02/16/2023, 06:07 PM PST
- vmaineng issue closed by PR 3978: https://github.com/hackforla/website/issues/3110#event-8542861641 at 02/16/2023, 06:07 PM PST
- vmaineng assigned: https://github.com/hackforla/website/issues/3922#issuecomment-1428972228 at 02/20/2023, 05:02 PM PST
- vmaineng commented on: https://github.com/hackforla/website/issues/3922#issuecomment-1437720862 at 02/20/2023, 05:04 PM PST
- vmaineng commented on: https://github.com/hackforla/website/pull/4017#issuecomment-1437729418 at 02/20/2023, 05:17 PM PST
- vmaineng commented on: https://github.com/hackforla/website/pull/4017#issuecomment-1437742422 at 02/20/2023, 05:35 PM PST
- vmaineng unassigned: https://github.com/hackforla/website/issues/3922#issuecomment-1437720862 at 02/21/2023, 11:29 AM PST
- vmaineng commented on: https://github.com/hackforla/website/pull/4017#issuecomment-1439186193 at 02/21/2023, 02:42 PM PST
- vmaineng issue closed as completed: https://github.com/hackforla/website/issues/3683#event-8609047083 at 02/25/2023, 11:21 PM PST
- vmaineng commented on: https://github.com/hackforla/website/pull/4099#issuecomment-1456982349 at 03/06/2023, 12:57 PM PST
- vmaineng submitted review: https://github.com/hackforla/website/pull/4099#pullrequestreview-1327223395 at 03/06/2023, 01:31 PM PST
